typescript-conditionals/main.ts

```diff
@@ -0,0 +1,103 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+function isUnderFive(number: number): boolean {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number: number): boolean {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(string: string): boolean {
+  if (string[0] === 'J' || string[0] === 'j') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+interface Person {
+  name: string;
+  age: number;
+}
+
+function isOldEnoughToDrink(person: Person): boolean {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrive(person: Person): boolean {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person: Person): boolean {
+  if (person.age > 0) {
+    return false;
+  } else {
```

Comment:
Can a person drink or drive if they are just older than 0?

Reply:
I originally made the function return false right away but it gave me an error for not using the `person` parameter. I wrote the `if` statement to use the parameter and still return false no matter what.

Reply:
While yes, it is true that the function should return `false` all of the time, the function should still do what it is intended to do: check if the person is old enough to drink and if the person is old enough to drive.